WebRoot/js/zimbraMail/calendar/controller/ZmCalViewController.js

```diff
@@ -1072,7 +1072,8 @@ function(buttons) {
 		ZmOperation.WORK_WEEK_VIEW,
 		ZmOperation.WEEK_VIEW,
 		ZmOperation.MONTH_VIEW,
-		ZmOperation.CAL_LIST_VIEW];
+		ZmOperation.CAL_LIST_VIEW,
+		ZmOperation.FB_VIEW];
```

Comment:
Will this make sure to show Free/Busy link in toolbar only when it is configured?

Reply:
Yes.  ZmCalViewController constructor function takes care of that. 
https://screencast.com/t/ANNUtWnajmPI